Depending on rest-assured (in some particular way, maven-dependency-plugin with <useRepositoryLayout>true?) fails Maven build under Java 11

The PR #1929 (mine) failed to build with this interesting message:

```
[WARNING] Failed to build parent project for com.sun.xml.bind.mvn:jaxb-bundles:pom:2.2.10
[WARNING] Failed to build parent project for com.sun.xml.bind.mvn:jaxb-osgi:pom:2.2.10
[WARNING] Failed to build parent project for com.sun.xml.bind:jaxb-osgi:jar:2.2.10
(...)
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-dependency-plugin:3.1.1:copy-dependencies (copy-dependencies) on project quarkus-gradle-plugin-integration-test: Some problems were encountered while processing the POMs:
[ERROR] [WARNING] Reporting configuration should be done in <reporting> section, not in maven-site-plugin <configuration> as reportPlugins parameter. @ 
[ERROR] [ERROR] 'dependencyManagement.dependencies.dependency.systemPath' for com.sun:tools:jar must specify an absolute path but is ${tools.jar} @ com.sun.xml.bind.mvn:jaxb-parent:[unknown-version], /home/vsts/.m2/repository/com/sun/xml/bind/mvn/jaxb-parent/2.2.10/jaxb-parent-2.2.10.pom, line 188, column 29: 2 problems were encountered while building the effective model for com.sun.xml.bind:jaxb-osgi:2.2.10
[ERROR] [WARNING] Reporting configuration should be done in <reporting> section, not in maven-site-plugin <configuration> as reportPlugins parameter. @ 
[ERROR] [ERROR] 'dependencyManagement.dependencies.dependency.systemPath' for com.sun:tools:jar must specify an absolute path but is ${tools.jar} @ com.sun.xml.bind.mvn:jaxb-parent:[unknown-version], /home/vsts/.m2/repository/com/sun/xml/bind/mvn/jaxb-parent/2.2.10/jaxb-parent-2.2.10.pom, line 188, column 29
```

It had passed when I locally built using Java 8 - this is a Java 11 specific build problem (reproducibly locally when building that PR with Java 11).

I'll work around it in that PR, but wanted to raise this issue for discussion whether we may would like to add some suitable `<exclusion>` (?) in some Quarkus POM?

--- a/bom/pom.xml
+++ b/bom/pom.xml
@@ -698,6 +698,11 @@
                         <groupId>javax.activation</groupId>
                         <artifactId>activation</artifactId>
                     </exclusion>
+                    <exclusion>
+                        <!-- https://github.com/quarkusio/quarkus/issues/1991 -->
+                        <groupId>com.sun.xml.bind</groupId>
+                        <artifactId>jaxb-osgi</artifactId>
+                    </exclusion>
                 </exclusions>
             </dependency>
 

